dipy/viz/streamline.py

```diff
@@ -1,13 +1,13 @@
 from dipy.utils.optpkg import optional_package
-
+import matplotlib.pyplot as plt
```

Comment:
this is an optional dependency, that's why CI's are failing, can you update this ? Thank you